include/board_pico_config.h

```diff
@@ -39,14 +39,22 @@
 #define PROBE_PIN_RESET 1
 #endif
 
-// UART config
+// UART config, ttyACMn
 #define PICOPROBE_UART_TX 4
 #define PICOPROBE_UART_RX 5
 #define PICOPROBE_UART_INTERFACE uart1
 #define PICOPROBE_UART_BAUDRATE 115200
 
+#if (CDC_UARTS > 1)
+// if enabled, always ttyACM(n+1)
+#define PICOPROBE_EX_UART_TX 0
```

Comment:
Not immediately obvious what `EX` stands for here - could be `EXternal`, `EXtra`, `EXtended`, `EXpired` ? :wink:

Reply:
was originally EXTRA, but just seemed too long - let me know preferred